plotly/plotly/plotly.py

```diff
@@ -1342,6 +1345,114 @@ def get_grid(grid_url, raw=False):
     return Grid(parsed_content, fid)
 
 
+class dashboard_ops:
+    """
+    Interface to Plotly's Dashboards API.
+    Plotly Dashboards are JSON blobs. They are made up by a bunch of
+    containers which contain either empty boxes or boxes with file urls.
+    """
```

Comment:
It'd be nice to include one or two simple getting started examples.

Ideally, the user has enough info when calling `help()` on all of these methods to get started. Here's what `help` looks like now:

![image](https://cloud.githubusercontent.com/assets/1280389/23555657/c2b90a94-fff7-11e6-8919-d3863f671c71.png)